AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/ItineraryController.cs

```diff
@@ -128,7 +127,7 @@ public ItineraryController(IMediator mediator, IItineraryEditModelValidator itin
         [Route("Admin/Itinerary/{id}/[Action]")]
         public async Task<IActionResult> SelectRequests(int id)
         {
-            var orgId = await _mediator.SendAsync(new OrganizationIdQuery { ItineraryId = id });
+            var orgId = await GetOrganizationIdBy(id);
```

Comment:
No this is fine.  You had just said _"(which I've now encapsulated in a private method with latest version of code, my PR still waiting to be merged)"_ and I thought you were referring to a different PR.  Apologies for my confusion